manager/manager.py

```diff
@@ -214,28 +210,37 @@ def start_job(self, job_id):
             self.update_job(job)
 
             payload = {
-                'data': {
-                    'job': job.get_db_data(),
-                }
+                'job': job.get_db_data(),
             }
-            print("Payload to {0}:".format(module.name))
-            print(payload)
 
-            job.log_message('Telling module {0} to convert {1} and put at {2}'.format(job.converter_module,
+            job.log_message('Telling module {0} to convert {1} and put at {2}'.format(module.name,
                                                                                       job.source,
                                                                                       job.output))
-            response = self.lambda_handler.invoke(module.name, payload)
+
+            headers = {"content-type": "application/json"}
+            url = module.public_links[0]
+            print("Payload to {0}:".format(url))
+            print(json.dumps(payload))
+            response = requests.post(url, json=payload, headers=headers)
+            print('finished.')
 
             print("Response from {0}:".format(module.name))
             print(response)
 
-            if 'errorMessage' in response:
-                job.error_message(response['errorMessage'])
-            elif 'Payload' in response:
-                payload = json.loads(response['Payload'].read())
+            json_data = response.json()
+            if json_data:
+                json_data = response.json()
+                if 'errorMessage' in json_data:
```

Comment:
I also decided to do the return of requests.post() a little different, in that I just set what should be returned in the test function, rather than have a mock_requests_posts_good and mock_requests_post_bad